src/components/calendar/calendar-rendering.spec.ts

```diff
@@ -140,19 +140,16 @@ describe('Calendar Rendering', () => {
       expect(last.children.item(6)?.part.contains('hidden')).to.be.true;
     });
 
-    it('should change orientation', async () => {
-      const dom = getCalendarDOM(calendar);
-
-      expect(
-        getComputedStyle(dom.content).getPropertyValue('flex-direction')
-      ).to.equal('row');
-
+    it('should set vertical orientation part', async () => {
       calendar.orientation = 'vertical';
       await elementUpdated(calendar);
 
-      expect(
-        getComputedStyle(dom.content).getPropertyValue('flex-direction')
-      ).to.equal('column');
+      const contentEl = calendar.shadowRoot?.querySelector('[part~="content"]');
+      expect(contentEl).to.exist;
+
+      const partAttr = contentEl?.getAttribute('part') || '';
+      const parts = partAttr.split(/\s+/);
+      expect(parts).to.include('content-vertical');
```

Comment:
If you update the `getCalendarDOM` function to correctly return the **content** container you can simplify this to:
```suggestion
      expect(dom.content).to.exist;
      expect(dom.content.part.contains('content-vertical')).to.be.true;
```